src/components/Common/TableOld/TableHead/TableHead.js

```diff
@@ -229,7 +226,20 @@ export const TableHead = observer(
           });
           return orderedColumns;
         }, [columns]);
-  
+        
+        useEffect(() => {
+          ref.current?.addEventListener("mousedown", (event) => {
+            if (event.target.className.includes("handle")) event.preventDefault();
+          });
+
+          return () => {
+            ref.current?.removeEventListener("mousedown", (event) => {
```

Comment:
this second parameter is useless because that's different function and it'll do nothing. also the whole block will be removed, so no need to unlisten

Reply:
correct! removing